lib/parser/char-block.cc

```diff
@@ -0,0 +1,18 @@
+//===-- lib/parser/char-block.cc --------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//----------------------------------------------------------------------------//
+
+#include "char-block.h"
+#include <ostream>
```

Comment:
Is ostream required here? Isn't it available through iosfwd in char-block.h? Alternatively, iosfwd can be removed from char-block.h.

Reply:
Oddly, ostream _is_ necessary here. I wasn't able to work out why, but it doesn't work with just isofwd which also surprised me. It seems that isofwd is missing the forward declaration of operator<<(std::string)... at least in libstdc++.
I can remove isofwd from char-block.h though, I don't think that's necessary anymore.

Reply:
OK. if it is not needed anymore in char-block.h then remove isofwd.